### PR TITLE
【增加】stm32f469-st-disco开发板的BSP

### PR DESCRIPTION
**Pull request by willianchanlovegithub**
Signed-off-by: Willian Chan <chentingwei@rt-thread.com>

## 拉取/合并请求描述：(PR description)

[
增加 stm32f469-st-disco 开发板的 BSP。
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other style
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality

### CODE REVIEW

**Comment by BernardXiong** [comment]
请签署CLA，并在PR页面上仔细对比提交的代码情况，然后在PR message中逐项打钩确认

File: bsp/stm32/stm32f469-st-disco/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+3. 输入`pkgs --update`命令更新软件包。
+
+4. 输入`scons --target=mdk4/mdk5/iar` 命令重新生成工程。
+
+本章节更多详细的介绍请参考 [STM32 系列 BSP 外设驱动使用教程](../docs/STM32系列BSP外设驱动使用教程.md)。
+
+## 注意事项
+
+- STM32F469NIH6U 这款芯片，RAM大小为 384KB，但是这 384KB 中有 64KB 是 CCM data RAM。在修改 `board.h` 文件和链接脚本文件的时候不能以 384KB 来配置，而应该以 (348 - 64)KB 即 320KB 来配置。
```
Comment (tyustli) [inline bsp/stm32/stm32f469-st-disco/README.md]: 这里RAM大小计算请重新计算

File: bsp/stm32/stm32f469-st-disco/rtconfig.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    LINK = PREFIX + 'gcc'
+    TARGET_EXT = 'elf'
+    SIZE = PREFIX + 'size'
+    OBJDUMP = PREFIX + 'objdump'
+    OBJCPY = PREFIX + 'objcopy'
+
+    DEVICE = ' -mcpu=cortex-m4 -mthumb -mfpu=fpv4-sp-d16 -mfloat-abi=hard -ffunction-sections -fdata-sections'
+    CFLAGS = DEVICE + ' -Dgcc'
+    AFLAGS = ' -c' + DEVICE + ' -x assembler-with-cpp -Wa,-mimplicit-it=thumb '
+    LFLAGS = DEVICE + ' -Wl,--gc-sections,-Map=rt-thread.map,-cref,-u,Reset_Handler -T board/linker_scripts/link.lds'
```
Comment (tyustli) [inline bsp/stm32/stm32f469-st-disco/rtconfig.py]: 请注意这里rtthread中间没有"-"

File: bsp/stm32/stm32f469-st-disco/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# STM32F469-ST-DISCO 开发板 BSP 说明
+
+## 简介
+
+本文档是石河子大学陈庭伟同学为 STM32F469-ST-DISCO 开发板提供的 BSP (板级支持包) 说明。
```
Comment (BernardXiong) [inline bsp/stm32/stm32f469-st-disco/README.md]: 陈庭伟同学加油！ 🥇

**Pull request merged**